### ISSUE TITLE
Alternative data structure

### ISSUE THREAD

**Issue by Coriou** [CSSEGISandData/COVID-19#207]
I'm using this data to generate a couples of graphs & maps for a bot of mine. I'm parsing the data and agglomerating it in a slightly easier way (country level only).

Just in case, you're stumbling upon this repo like I did, and just want to wrap up something quick, you can check this out: https://api.jsonbin.io/b/5e5a4b3c09ac43054814a7b8/latest

The data is compiled from those different files:

- [csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Confirmed.csv](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Confirmed.csv)
- [csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Deaths.csv](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Deaths.csv)
- [csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Recovered.csv](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Recovered.csv)
- [csse_covid_19_data/csse_covid_19_daily_reports/](https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports)

No other data is used. It's updated at least twice a day. I just thought I'd share because it might help someone. Would have helped me :)

Here's an excerpt:

```json
{
  "lastUpdated": "2020-03-03T18:42:07.286Z",
  "latestUpdate": "2020-03-02T22:53:02.000Z",
  "currentCases": 90298,
  "currentDeaths": 3084,
  "currentHealed": 45596,
  "currentStillInfected": 41618,
  "countryData": [
    {
      "country": "France",
      "lat": 47,
      "lon": 2,
      "deaths": [
        {
          "date": "2020-01-22T00:00:00.000Z",
          "value": 0
        },
       ...
      ],
      "cases": [
        {
          "date": "2020-01-22T00:00:00.000Z",
          "value": 0
        },
        ...
      ],
      "healed": [
        {
          "date": "2020-01-22T00:00:00.000Z",
          "value": 0
        },
        ...
      ],
      "currentDeaths": 3,
      "currentCases": 191,
      "currentHealed": 12,
      "lastUpdated": "2020-03-02T19:33:02.000Z",
      "stillContaminated": 176,
      "healRatio": 6.282722513089005,
      "deathRatio": 1.5706806282722512
    },
    ...
  ]
}
```

**Comment by MehranDHN** [comment]
> I'm using this data to generate a couple of graphs & maps for a bot of mine. I'm parsing the data and agglomerating it in a slightly easier way (country level only).
> 
> Just in case, you're stumbling upon this repo like I did, and just want to wrap up something quick, you can check this out: https://api.jsonbin.io/b/5e5a4b3c09ac43054814a7b8/latest
> 
> The data is compiled from those different files:
> 
> * [csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Confirmed.csv](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Confirmed.csv)
> * [csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Deaths.csv](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Deaths.csv)
> * [csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Recovered.csv](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_19-covid-Recovered.csv)
> * [csse_covid_19_data/csse_covid_19_daily_reports/](https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports)
> 
> No other data is used. It's updated at least twice a day. I just thought I'd share because it might help someone. Would have helped me :)
> 
> Here's an excerpt:
> 
> ```json
> {
>   "lastUpdated": "2020-03-03T18:42:07.286Z",
>   "latestUpdate": "2020-03-02T22:53:02.000Z",
>   "currentCases": 90298,
>   "currentDeaths": 3084,
>   "currentHealed": 45596,
>   "currentStillInfected": 41618,
>   "countryData": [
>     {
>       "country": "France",
>       "lat": 47,
>       "lon": 2,
>       "deaths": [
>         {
>           "date": "2020-01-22T00:00:00.000Z",
>           "value": 0
>         },
>        ...
>       ],
>       "cases": [
>         {
>           "date": "2020-01-22T00:00:00.000Z",
>           "value": 0
>         },
>         ...
>       ],
>       "healed": [
>         {
>           "date": "2020-01-22T00:00:00.000Z",
>           "value": 0
>         },
>         ...
>       ],
>       "currentDeaths": 3,
>       "currentCases": 191,
>       "currentHealed": 12,
>       "latestUpdate": "2020-03-02T19:33:02.000Z",
>       "stillContaminated": 176,
>       "healRatio": 6.282722513089005,
>       "deathRatio": 1.5706806282722512
>     },
>     ...
>   ]
> }
> ```

Does it restricted by CORS?

**Comment by Coriou** [comment]
> Does it restricted by CORS?

Yes. The service I use to host the JSON is jsonbin.io (not affiliated) and you can read about it there: https://jsonbin.io/api-reference

**Comment by Coriou** [comment]
JSONBIN won't allow me to post there anymore, I think the payload got too big. I've reached out to them to see if this was the case.

The latest data can now be found as a Gist at https://gist.github.com/Coriou/386e3527aa5b1d5409d4397c48141e5e
